task_sdk/tests/conftest.py

```diff
@@ -16,10 +16,15 @@
 # under the License.
 from __future__ import annotations
 
+import os
+
 import pytest
 
 pytest_plugins = "tests_common.pytest_plugin"
 
+# Task SDK does not need access to the Airflow database
+os.environ["_AIRFLOW_SKIP_DB_TESTS"] = "true"
```

Comment:
I believe we’re running the Task SDK tests in a separate process, as indicated by the dedicated task-sdk-tests.yml GitHub workflow. However, I hope that we aren’t running any other tests/modules in the same process elsewhere that could be impacted by this setup.

Reply:
Pytest won't let you run yet unrelated test folders - i.e. ones that don't share a conftest.py in parent dir, and some there isn't a one at the top level of the repo we're good